pkgs/applications/misc/ff2mpv/default.nix

```
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
```

Hope you don't mind some extra suggestions, since it's a refactor and all.

```suggestion
  installPhase = ''
    runHook preInstall
```

---

Done.